letsencrypt_plugin.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.1.0'
 
-  s.add_dependency 'rails', '>= 4.2', '< 5.1'
+  s.add_dependency 'rails', '>= 4.2'
```

Comment:
That's called "upper bound pessimism" and is [not recommended](https://github.com/rails/rails-dom-testing/commit/faf6f6959fb00d6627ff88b2a35f57eb55a35655). 

Because it does not add any benefit, rather it introduces **unneeded** problems each time a new Rails version is released. This is exactly what happened when Rails `5.0` was introduces here: https://github.com/lgromanowski/letsencrypt-plugin/issues/35 and caused trouble for a number of developers.

Are you aware of any benefit to your approach? I'm not, but I'm willing to learn :)